integration-tests/pom.xml

```
@@ -91,6 +91,11 @@
             <artifactId>maven-surefire-plugin</artifactId>
             <configuration>
               <skipTests>${native.surefire.skip}</skipTests>
+              <environmentVariables>
+                <NAMESPACE_FROM_ENV>fromEnvVarNS</NAMESPACE_FROM_ENV>
```

If you want to make sure both Surefire and Failsafe are consistent, you can probably do:
```suggestion
                <NAMESPACE_FROM_ENV>${env.NAMESPACE_FROM_ENV}</NAMESPACE_FROM_ENV>
```
But I would advise you to do it in a second step. Let's keep it simple first and iterate with simple stuff to understand how we can pass the values.